curriculum/challenges/english/25-front-end-development/quiz-relational-database/66f1af82732957c895f0b21a.md

```diff
@@ -17,439 +17,440 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --distractors--
 
-Placeholder distractor 1
+Non-relational databases use tables, while relational databases do not
 
 ---
 
-Placeholder distractor 2
+Relational databases can store unstructured data
 
 ---
 
-Placeholder distractor 3
+Non-relational databases are slower than relational databases
 
 #### --answer--
 
-Placeholder answer
+Non-relational databases can store unstructured data
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the PostgreSQL command \l do?
 
 #### --distractors--
 
-Placeholder distractor 1
+Lists all schemas
 
 ---
 
-Placeholder distractor 2
+Lists all tables
 
 ---
 
-Placeholder distractor 3
+Connects to a database
 
 #### --answer--
 
-Placeholder answer
+Lists all databases
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following commands creates a new table in SQL?
 
 #### --distractors--
 
-Placeholder distractor 1
+ADD TABLE
 
 ---
 
-Placeholder distractor 2
+NEW TABLE
 
 ---
 
-Placeholder distractor 3
+ALTER TABLE
 
 #### --answer--
 
-Placeholder answer
+CREATE TABLE
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the purpose of a foreign key in a relational database?
 
 #### --distractors--
 
-Placeholder distractor 1
+To allow duplication of data
 
 ---
 
-Placeholder distractor 2
+To uniquely identify a row in the table
 
 ---
 
-Placeholder distractor 3
+To store large amounts of text data
 
 #### --answer--
 
-Placeholder answer
+To create a relationship between two tables
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command will add a new column to an existing table?
 
 #### --distractors--
 
-Placeholder distractor 1
+MODIFY TABLE
 
 ---
 
-Placeholder distractor 2
+INSERT COLUMN
 
 ---
 
-Placeholder distractor 3
+ADD COLUMN
 
 #### --answer--
 
-Placeholder answer
+ALTER TABLE
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What type of relationship exists when one record in Table A corresponds to one or more records in Table B?
 
 #### --distractors--
 
-Placeholder distractor 1
+One-to-One (1-1)
 
 ---
 
-Placeholder distractor 2
+Many-to-Many (m-m)
 
 ---
 
-Placeholder distractor 3
+Many-to-one(m-1)
 
 #### --answer--
 
-Placeholder answer
+One-to-Many (1-m)
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will the following SQL query return?
 
 #### --distractors--
 
-Placeholder distractor 1
+Count of employee names
 
 ---
 
-Placeholder distractor 2
+List of all employee IDs
 
 ---
 
-Placeholder distractor 3
+Average salary of all employees
 
 #### --answer--
 
-Placeholder answer
+List of all employee names
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How do you delete a column from a table in SQL?
 
 #### --distractors--
 
-Placeholder distractor 1
+REMOVE COLUMN
 
 ---
 
-Placeholder distractor 2
+DELETE COLUMN
 
 ---
 
-Placeholder distractor 3
+ALTER TABLE
 
 #### --answer--
 
-Placeholder answer
+DROP COLUMN
```

Comment:
This answer is not consistent with answer to question regarding adding new column. Although both have part of the command, instead of complete one.

Reply:
Changes Done
Please review my code!